packages/sit-onyx/src/components/OnyxColorSchemeDialog/OnyxColorSchemeDialog.stories.ts

```diff
@@ -0,0 +1,32 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxColorSchemeDialog from "./OnyxColorSchemeDialog.vue";
+
+/**
+ * Pre-built dialog where the user can select which color scheme (light/dark mode or auto) to use for the application.
+ *
+ * For recommended usage, please visit our [theming docs](https://onyx.schwarz/development/theming.html).
+ */
+const meta: Meta<typeof OnyxColorSchemeDialog> = {
+  title: "support/ColorSchemeDialog",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxColorSchemeDialog,
+    events: ["update:modelValue", "close"],
+    decorators: [
+      (story) => ({
+        components: { story },
+        template: `<div style="height: 48rem;"> <story /> </div>`,
+      }),
+    ],
+  }),
+};
+
+export default meta;
+type Story = StoryObj<typeof OnyxColorSchemeDialog>;
+
+export const Default = {
```

Comment:
the docs page for this example is problematic.
http://localhost:6006/?path=/docs/support-colorschemedialog--docs

I'd recommend to use a "show dialog" button in the example and connect the cancel/apply to closing the example?
like we did in the examples of https://core-ui.schwarz/components/scu-modal-dialogue